packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -86,4 +87,41 @@ test.describe('Participant pane tests', async () => {
 
     expect(await stableScreenshot(page)).toMatchSnapshot(`PSTN-call-screen-with-dialpad.png`);
   });
+
+  test.only('callee participant is displayed with connection state', async ({ page, serverUrl }) => {
+    test.skip(isTestProfileStableFlavor());
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
```

Comment:
@dmceachernmsft recently added a better helper to create 1:N and PSTN remote participants. Please use one of those so that the state is more realistic.

Reply:
Doesn't matter, we are just testing the names here.

Reply:
there isn't much of a risk but I think the idea here is for readability. I think we should be able to read the tests and know exactly what is happening and using the kind of participant as PSTN could help us determine what kind of call it is.